setup.py

```diff
@@ -42,7 +42,6 @@
         'transaction',
         'Persistence',
         'zExceptions',
-        'ZODB3',
```

Comment:
Maybe, but nobody using Zope 2.13.x will bill in Products.ZCTextIndex 2.14.x, so the overlap should be effectively nil.